### ISSUE TITLE
kdenlive: build broken

### ISSUE THREAD

**Issue by zraexy** [NixOS/nixpkgs#26160]
## Issue description

Kdenlive build fails because CMake can't find QtConcurrent.

Error:
```
CMake Error at /build/__nix_qt5__/lib/cmake/Qt5Core/Qt5CoreMacros.cmake:330 (message):
  Can not use "Concurrent" module which has not yet been found.
Call Stack (most recent call first):
  renderer/CMakeLists.txt:12 (qt5_use_modules)
```

It appears to have been broken by some changes made to qtbase but I am not familiar enough with it to know what specifically.

Changes between success and first failure: https://hydra.nixos.org/api/scmdiff?type=git&branch=&rev2=f099f55e5b9e9bb39db7662b6f1b09000b05d369&rev1=fa5196e47e3a72553155f684c8e66a97d91e6dff&uri=https%3A%2F%2Fgithub.com%2FNixOS%2Fnixpkgs.git

Latest Hydra failure: https://hydra.nixos.org/build/53342055

cc @ttuegel 

### Steps to reproduce

Attempt to install Kdenlive

## Technical details

* System: NixOS: 17.09pre107928.65477dcd34 (Hummingbird)
* Nix version: nix-env (Nix) 1.11.9
* Nixpkgs version: "17.09pre107928.65477dcd34"
* Sandboxing enabled: build-use-sandbox = true

**Comment by bkchr** [comment]
You need to modify the CMakeLists.txt:
`find_package(Qt5 REQUIRED COMPONENTS Core DBus Widgets Script Svg Quick )` to
`find_package(Qt5 REQUIRED COMPONENTS Core DBus Widgets Script Svg Quick Concurrent)`

We should send a patch upstream. This must be a required change since QT5.8.

**Comment by zraexy** [comment]
Fixed by: dc073726927f749116d2a7c0e8257723a76cf603
